app/onepassword_events_api/default/data/ui/views/item_usage_dashboard.xml

```diff
@@ -89,7 +89,7 @@
       <chart>
         <title>Top Item Users</title>
         <search>
-          <query>index=$index$ sourcetype="1password:insights:item_usages" |  stats count by user.email | sort - count</query>
+          <query>index=$index$ sourcetype="1password:insights:item_usages" | stats count by user.email | sort - count</query>
```

Comment:
I think it would be valuable to have `user.email` be customizable here as well

Reply:
++ good call!